Create writeFirestore.js

jlbee opened the pull request:
Template methods for uploading content. Variables will need to be added.

jlbee committed:
Create writeFirestore.js

Template methods for uploading content. Variables will need to be added.

kvmercer commented on firebase/firestore/writeFirestore.js:
```diff
@@ -0,0 +1,102 @@
+    
+    */
+    
+    //Adding questions. Would depend on above. 
```
i would say comment this out as well, were not dealing with quizzes yet, but this will bed good to come back to :)

Commit:
cleanup

jlbee committed:
Create writeTest.js

ryan-hong commented on firebase/firestore/writeFirestore.js:
```diff
@@ -0,0 +1,106 @@
+                    .firestore()
+                    .collection('Announcements')
+                    .doc(CourseCode) 
+                    .set({
```
can we have 
`title:
time:
course:
content:
`
instead? just matches and makes more sense with my admin code

ryan-hong replied:
as params i mean, so content instead of paragraph and just add another field for course

jlbee committed:
Update writeFirestore.js

Commented out quiz stuff, and added an extra field for course code in the course announcement.
